### ISSUE TITLE
test_field_statistics fails Komodo testing

### ISSUE THREAD

**Issue by berland** [equinor/subscript#745]
https://github.com/equinor/komodo-releases/actions/runs/11116956374/job/30888414795

```
    def make_filestream(filelike):
>       file_stream = open(filelike, "rb")  # noqa SIM115
E       FileNotFoundError: [Errno 2] No such file or directory: '/mnt/scratch/f_scout_ci/actions-runner-02/_temp/pytest_tmp_dir/pytest-of-f_scout_ci/pytest-0/popen-gw5/test_main_config_file0_config_0/testdata_field_statistics/ensemble/share/grid_statistics/ertbox--prob_A_F3_0.roff'

/prog/komodo/bleeding-20241001-0330-py38-rhel8/root/lib64/python3.8/site-packages/_roffio/reading.py:43: FileNotFoundError
```

**Comment by mferrera** [comment]
Should be fixed by #744

**Comment by berland** [comment]
Will this be included in subscript's own CI as well?

**Comment by mferrera** [comment]
It should be included by default. I am speculating (I was not part of reviewing the code) that there is a concurrency issue with the tests from pytest-xdist that hits on Komodo and not here. Probably because action runners only have two cores to distribute over (or just luck).

**Comment by alifbe** [comment]
From what I can see, Komodo is probably putting `src` and `tests` in different setup compare to subscript. The test assume the folder structure during testing, which is not needed. There is a fix in PR #747 commit https://github.com/equinor/subscript/pull/747/commits/260ce1dbd13376ddc79efc37f1c904d81070f1b7
https://github.com/equinor/subscript/blob/b5b3861e0e84ff08637b049feb1b930ed6e14452/tests/test_field_statistics.py#L896-L897

**Comment by berland** [comment]
> From what I can see, Komodo is probably putting `src` and `tests` in different setup compare to subscript. 

That is correct, details here: https://github.com/equinor/komodo/blob/main/ci/func_def.sh

**Comment by alifbe** [comment]
Fixed in #747
